### PR TITLE
Wes review 2024-07-15

### PR DESCRIPTION
**Pull request by hardaker**

### CODE REVIEW

**Review by dhruvdhody — CHANGES_REQUESTED** [review]
Can we break this into 2 sentences -

Collaboratively sketch new requirements for future network management operations with the industry, network operators, and protocol engineers. Develop a plan of action and recommendations for the IETF.

**Comment by dhruvdhody** [comment]
Thanks Wes! Except for that one very minor change request, it is ready to merge. Thanks!

File: IAB-Workshop.md [hunk 1]
```diff
@@ -31,33 +32,35 @@ Regarding operator deployments, discussion topics will include, but are not limi
...
 
-Under this effort, we expect to stimulate an outreach effort to attract opinions and interest by visiting these other venues and running virtual sessions. This series of outreach efforts will be concluded by a final online Workshop where the group will summarize the findings. 
+Under this effort, we expect some workshop participants to stimulate an outreach effort to attract opinions and interest by visiting these other venues and running information and requirement gathering sessions. This series of outreach efforts will result in workshop position papers that will provide valuable summarized feedback to the online NEMOPS Workshop. 
 
-The mailing list nemops-interest@iab.org will be used for outreach events at other conferences, general announcements, and related discussions before the workshop.
+The mailing list nemops-interest@iab.org will be used for outreach events at other conferences, general announcements, and related discussions before (and after) the workshop is held at the end of 2024.
```
Comment (boucadair) [inline IAB-Workshop.md:63]:
```suggestion
The mailing list nemops-interest@iab.org will be used for outreach events at other conferences, general announcements, and related discussions before (and after) the workshop.
```

File: IAB-Workshop.md [hunk 2]
```diff
@@ -6,18 +6,19 @@ The IAB organized a workshop in June 2002 to establish a dialog between network
...
-
-More than 20 years later, it is timely to evaluate what has been achieved since the original workshop. This new workshop aims to discuss the following key topics:
+It has been more than 20 years since that IAB workshop and it is time
+to evaluate what has been achieved since the original workshop and
+what recommendations and requirements have changed or still need pursing. This new workshop aims to discuss the following key topics:
 
 - Review the outcomes and results of the 2002 workshop (current deployments, state of the art) and identify any operational barriers that prevent these technologies from being widely implemented (limitations, hurdles).
 
-- Sketch new requirements for future network management operations in a collaborative manner with the industry, and develop a plan of action and recommendations for the IETF.
+- Sketch new requirements for future network management operations in a collaborative manner with the industry, network operators and protocol engineers and develop a plan of action and recommendations for the IETF.
```
Comment (boucadair) [inline IAB-Workshop.md:15]:
```suggestion
- Sketch new requirements for future network management operations in a collaborative manner with the industry, network operators, and protocol engineers.

 - Develop a plan of action and recommendations for the IETF.
```

File: IAB-Workshop.md [hunk 3]
```diff
@@ -6,18 +6,19 @@ The IAB organized a workshop in June 2002 to establish a dialog between network
...
 
 ## Review of the 2002 Workshop Outcomes
 
-The IAB NEMOPS workshop will provide an assessment of the RFC3535 requirements in terms of specifications completeness. Also, this workshop will assess to what extent the RFC3535 recommendations were driving network management efforts within the IETF and influencing other Standards Development Organizations' (SDOs) activities. 
+The first focus of the IAB NEMOPS workshop will be to assess the recommendations and requirements documented in RFC3535 in terms of specification completeness, deployment and utilization. Also, this workshop will assess to what extent the RFC3535 recommendations were driving network management efforts within the IETF and influencing other Standards Development Organizations' (SDOs) activities.
```
Comment (boucadair) [inline IAB-Workshop.md:19]:
```suggestion
The first focus of the IAB NEMOPS workshop will be to assess the requirements documented in RFC3535 in terms of specification completeness, deployment, and utilization. Also, this workshop will assess to what extent the RFC3535 recommendations were driving network management efforts within the IETF and influencing other Standards Development Organizations' (SDOs) activities. 
```

Comment (boucadair) [reply IAB-Workshop.md:19]: 3535 reco are IETF-centric and are mentioned in the sentence right after.

File: IAB-Workshop.md (hunk 2 above)
Comment (boucadair) [reply IAB-Workshop.md:15]: Split the sentence for better readability

File: IAB-Workshop.md [hunk 4]
```diff
@@ -31,33 +32,35 @@ Regarding operator deployments, discussion topics will include, but are not limi
 
 - What alternative (i.e., non-IETF) solutions are being deployed that better fit operators' needs?
 
+- What recommendations and requirements from RFC3535 have received either or both little implementation or deployment?
```
Comment (boucadair) [inline IAB-Workshop.md:35]:
```suggestion
- What requirements from RFC3535 have received either or both little implementation or deployment?
```

File: IAB-Workshop.md [hunk 5]
```diff
@@ -31,33 +32,35 @@ Regarding operator deployments, discussion topics will include, but are not limi
...
+- Tooling, opensource, experimentation, proof of concept, multi-vendor interoperability test (e.g., EANTC), and system integration
 
 - Data consistency to support richer observability (Data & Knowledge)
 
 - Integration issues with the business layer
 
 - Automation, orchestration, and autonomy 
 
-Recommendations that may come out of this effort should be helpful to the global industry, including the IETF, the IRTF, as well as implementers, operational groups, and other entities.
+Recommendations that may come out of this effort should be helpful to the standardization development organizations including the IETF and the IRTF, as well as the global industry's implementers, operational groups, and other entities.
```
Comment (boucadair) [inline IAB-Workshop.md:51]:
```suggestion
Recommendations that may come out of this effort should be helpful to the global industry including the IETF, the IRTF, implementers, operational groups, and other entities.
```

**Review by boucadair — CHANGES_REQUESTED** [review]
Thanks for the edits.

File: IAB-Workshop.md (hunk 3 above)
Comment (hardaker) [reply IAB-Workshop.md:21]: Those are two different notions in the sentences though.  So, IMHO, they should both say "recommendations and requirements" as the first sentence talks about compleness, deployment and utilization of both, and the second is about whether or not both have affected other SDOs or not.

File: IAB-Workshop.md (hunk 4 above)
Comment (hardaker) [reply IAB-Workshop.md:37]: disagree with this -- 3535 has both recommendations and requirements and I was trying to make it very clear that both will be discussed/analyzed/etc.  It's overly verbose, but such is the way of the IETF (IMHO).

File: IAB-Workshop.md (hunk 5 above)
Comment (hardaker) [reply IAB-Workshop.md:53]: So we don't intend to create advice that might be helpful to SDOs?

File: IAB-Workshop.md (hunk 3 above)
Comment (boucadair) [reply IAB-Workshop.md:21]: I'm having troubles to see what "deployment, and utilization" is about for a set of reco that were about creating WGs, dedicate resources, and some other IETF matters.

File: IAB-Workshop.md (hunk 5 above)
Comment (boucadair) [reply IAB-Workshop.md:53]: other "SDOs" fall under "other entities".  

Actually, the initial wording (and thus what this change tries to revert back) is to put the emphasis on the global industry approach not separating SDOs vs rest of the ecosystem.

File: IAB-Workshop.md (hunk 4 above)
Comment (boucadair) [reply IAB-Workshop.md:37]: Fully agree that both need to be discussed/analyzed/etc. 

My concern is related to mixing these two and talk about "deployment, etc." when referring to considerations related to IETF resources (3535 reco). Please note that we do already have the following in the preamble: 

"Assess to what extent the RFC3535 recommendations were driving network management efforts within the IETF"

File: IAB-Workshop.md (hunk 3 above)
Comment (billwuqin) [reply IAB-Workshop.md:21]: Have this issue resolved? My understanding on utilization is about whether specific part of recommendation or requirements have been quoted or referenced by some standard work in other SDOs.

File: IAB-Workshop.md (hunk 5 above)
Comment (billwuqin) [reply IAB-Workshop.md:53]: see SDOs as part of global industry is new to me but fine, traditionally SDOs focus on developing standard work while industry consortium is responsible for get it deployed or implemented.

File: IAB-Workshop.md (hunk 4 above)
Comment (hardaker) [reply IAB-Workshop.md:37]: I think we want to do both though -- we want to see what was done within the IETF, and whether the results of anything done inside the ietf were used outside.  IE, is netconf getting used?  is SNMP getting sunsetted?  Is everyone just switching to REST?  What's the state of the world, and is it diverging from IETF specs and what can the IETF do (now) to improve things.  It involves both the past, the effect internal and the effect externally, right?

File: IAB-Workshop.md (hunk 5 above)
Comment (bclaise) [reply IAB-Workshop.md:53]: So we don't intend to create advice that might be helpful to SDOs? => yes we do, but you will see that, getting in the higher levels of OPS architecture, more and more opensource will be involved ... where the IETF is the facilitator. Therefore, I agree with Med's proposal

**Comment by hardaker** [comment]
for any outstanding changes, since this is now merged, I'd create separate pull requests for them

**Pull request merged**